meter.go

```diff
@@ -0,0 +1,230 @@
+package otelpgx
+
+import (
+	"context"
+	"fmt"
+	"sync"
+	"time"
+
+	"github.com/jackc/pgx/v5/pgxpool"
+	"go.opentelemetry.io/otel"
+	"go.opentelemetry.io/otel/attribute"
+	"go.opentelemetry.io/otel/metric"
+	semconv "go.opentelemetry.io/otel/semconv/v1.27.0"
+)
+
+const (
+	// defaultMinimumReadDBStatsInterval is the default minimum interval between calls to db.Stats().
+	defaultMinimumReadDBStatsInterval = time.Second
+)
+
+var (
+	pgxPoolAcquireCount            = "pgxpool.acquires"
+	pgxPoolAcquireDuration         = "pgxpool.acquire_duration"
```

Comment:
I'm not so familiar with the best practices around metric naming here, but would it make sense to add the base unit?
```suggestion
	pgxPoolAcquireDuration         = "pgxpool.acquire_duration_seconds"
```

Reply:
As long as each metric specifies their unit upon creation, we shouldn't need to include them in the metric name ourselves.
https://opentelemetry.io/docs/specs/semconv/general/metrics/#units

To this end, I did realize I missed specifying the unit in the async. metric `pgxPoolAcquireDuration` in `meter.go`. This has now been fixed.

FWIW as well-- Depending on users' observability stack and its own configuration, it is possible for the metric unit to be added to the metric name later on.
Using Prometheus as an example, it will append `_milliseconds_bucket` to the metric name `db_client_operation_duration`.
https://github.com/exaring/otelpgx/blob/f84cbc7a6d2ba2abbd2cc1786b552ac6ef6d9f89/tracer.go#L136

![image](https://github.com/user-attachments/assets/ebcbbdf8-0979-40b0-93b5-d9fb90bae0d6)